app/components/main-hero/meetup/template.hbs

```diff
@@ -0,0 +1,15 @@
+<a
+  href="https://www.eventbrite.com/e/ember-fastboot-ssr-beyond-tickets-144468965565"
+  class="main-hero-meetup"
+  target="_blank"
+  rel="noopener">
+  <div class="content">
+    <p class="invite">
+      Joins us <span class="orange">&nbsp;Wed April 14th</span>
```

Comment:
Typo: Joins -> Join

Reply:
May be we should add in a time as well?

Reply:
I have fixed the typo and added a time. I know it is not beautiful but it is functional.